assets/src/edit-story/components/panels/design/preset/presets.js

```diff
@@ -37,8 +58,20 @@ function Presets({ presets, handleOnClick, itemRenderer, isEditMode, type }) {
           type={type}
           isEditMode={isEditMode}
           handleClick={handleOnClick}
+          handleAddPreset={handleAddPreset}
         />
       )}
+      {!hasPresets && 'color' === type && (
+        <ButtonWrapper>
+          <ColorAdd
+            handleAddPreset={handleAddPreset}
+            helper={__(
+              'Click on the + icon to save a color to all stories',
```

Comment:
Could we perhaps use the actual + icon here? (using `TranslateWithMarkup` for instance)

If that's not possible, maybe use `sprintf` and a placeholder instead of the `+` so it doesn't get translated accidentally?